lxqt-base/lxqt-config/metadata.xml

```diff
@@ -10,5 +10,6 @@
 	</upstream>
 	<use>
 		<flag name="monitor" restrict="&gt;=lxqt-base/lxqt-config-0.13.0">Build monitor settings tool</flag>
+		<flag name="touchpad" restrict="&gt;=lxqt-base/lxqt-config-0.13.0">Build touchpad settings tool</flag>
```

Comment:
this line and above: why the restrictions? it does not matter if the flags did not exist in prior versions, and they probably did not change their meaning otherwise.

Reply:
Good question.  It was already there when I got to it, but didn't really think if it's required or not.

I remember seeing it before, somewhere, and thinking it might have something to do with exactly that, other versions not having it... but I didn't research further back then either.

Will remove.